pkgs/development/python-modules/djhtml/default.nix

```
@@ -0,0 +1,21 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "djhtml";
+  version = "3.0.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-84/U59V1U4snkcXKorVMU5etwI+PjSZYU+Ha+oojmuk=";
+  };
+
+  meta = with lib; {
+    homepage = "https://github.com/rtts/djhtml";
+    description = "A pure-Python Django/Jinja template indenter without dependencies.";
```

```suggestion
    description = "Django/Jinja template indenter";
```

see https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes about the period at the end.